packages/shared/lib/core/ledger/interfaces/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './ledger-connection-state.interface'
 export * from './ledger-extended-status.interface'
 export * from './ledger-status.interface'
+export * from './ledger-send-confirmation-props.interface'
```

Comment:
Sometimes exporting in alphabetical order breaks compilation, was that also the case here?